tests/modules/civicrm_entity_test/migrations/civicrm_events.yml

```diff
@@ -0,0 +1,28 @@
+id: civicrm_entity_test_civicrm_event
+label: 'Import CiviCRM events for CiviCRM Entity Test'
+migration_group: 'CiviCRM Entity'
+migration_tags:
+  - 'CiviCRM Entity'
+source:
+  plugin: embedded_data
+  ids:
+    title:
+      type: string
+  data_rows:
+    - title: 'DrupalCon Nashville'
+    - summary: 'DrupalCon Nashville 2018! Get your Drupal on'
```

Comment:
The extra '-' in front of 'summary' here seems not right. If I'm reading this correctly, this would make two data rows like this:

```
[
  [
    'title' => 'DrupalCon Nashville',
  ],
  [
    'summary' => 'DrupalCon Nashvilee 2018! Get your Drupal on',
    'description' => '...',
    // etc
  ]
]
```